models/index.js

```diff
@@ -0,0 +1,30 @@
+const Sequelize = require('sequelize')
+const allConfigs = require('../configs/sequelize')
+const monthsModel = require('./months')
+const moonsModel = require('./moons')
+const activitiesModel = require('./activities')
+
+const environment = process.env.NODE_ENV ? process.env.NODE_ENV : 'development'
+const config = allConfigs[environment]
+
+const connection = new Sequelize('witches', 'witches', 'w!tChp1e@$3', {
+  host: 'localhost', dialect: 'mysql'
+})
```

Comment:
This should be reading from your config file so that when another developer (me) creates a .env file and tries to run your code it all works. You even go through the code to get the correct `config` variable and then leave it unused. This is why all of your routes failed when I tried them.